website/server/controllers/api-v4/inbox.js

```diff
@@ -93,4 +93,52 @@ api.conversations = {
   },
 };
 
+function mapMessage (newChat, user) {
```

Comment:
can this be moved to the message model?

Reply:
I tried the `schema.methods`-way but none the properties were changed, so I just moved the method to the same file but outside of the `schema.methods`